Pulls/1.0/unbreak my ci

NightJar opened the pull request:
Get travis mooving again.

robbieaverill commented on composer.json:
```diff
@@ -8,8 +8,8 @@
     "require": {
-        "cwp/cwp": "^2",
-        "silverstripe/fulltextsearch": "^3"
+        "cwp/cwp": "2.0.0-rc1",
```
I dislike this change

NightJar replied:
Hmm, ✔️ 
Was testing from tag, forgot to revert :/
(this is the `1.0.0-rc1` tag commit)

Commit:
FIX use appropriate constraints for our CI testing

This will prevent issues such as an unresolvable requirements list causing
the build to error. Currently this happens as the travis config pulls in
too tight a depenency constraint, which is already too outdated for our
testing needs.
